lib/jinx.ex

```diff
@@ -1,2 +1,44 @@
 defmodule Jinx do
+
+  @moduledoc """
+  Jinx is a collection of flexible helpers to help
```

Comment:
"Defines flexible test helpers for creating and testing JSON data [JSON APIs] (http://jsonapi.org)"?